blockchain/state/journal.go

```diff
@@ -140,10 +140,15 @@ type (
 	accessListAddAccountChange struct {
 		address *common.Address
 	}
+
 	accessListAddSlotChange struct {
 		address *common.Address
 		slot    *common.Hash
 	}
+	transientStorageChange struct {
```

Comment:
```suggestion
        // Changes to transient storage
	transientStorageChange struct {
```

Reply:
The name of the struct already describes the comment. Still, thanks for the suggestion.